web/packages/teleport/src/Roles/RoleEditor/StandardEditor/AdminRules.tsx

```diff
@@ -194,3 +213,86 @@ function ResourceKindMultiValue(props: MultiValueProps<ResourceKindOption>) {
     </HoverTooltip>
   );
 }
+
+/** Renders a grid of allowed permissions (verbs) as a grid of checkboxes. */
+function VerbEditor({
+  verbs,
+  allVerbs,
+  rule,
+  onVerbChange,
+  onAllVerbsChange,
+}: {
+  verbs: VerbModel[];
+  allVerbs: boolean;
+  rule: Rule<VerbModel[]>;
+  onVerbChange(verb: Verb, checked: boolean): void;
+  onAllVerbsChange(checked: boolean): void;
+}) {
+  const id = useId();
+  const { valid, message } = useRule(rule(verbs));
+
+  // This is dumb, but it works for now, as the editor is fixed-width.
+  const numColumns = verbs.some(
```

Comment:
where is the fix-width defined? Can you reference that here so it will be helpful to lookup if we want to change this in future.

Reply:
It's defined in the RoleEditorAdapter. Adding appropriate comments in both places.